internal/kafka/internal/services/data_plane_kafka.go

```diff
@@ -280,6 +282,26 @@ func (d *dataPlaneKafkaService) reassignKafkaCluster(kafka *dbapi.KafkaRequest)
 	return nil
 }
 
+func (d *dataPlaneKafkaService) unassignKafkaFromDataplaneCluster(kafka *dbapi.KafkaRequest) *serviceError.ServiceError {
+	if kafka.Status == constants2.KafkaRequestStatusProvisioning.String() {
+		if err := d.kafkaService.Updates(kafka, map[string]interface{}{
```

Comment:
can we add a log about that says that the Kafka is being unassigned from the current cluster.

Reply:
Thanks @machi1990 
That has been added, let me know if its ok :)